docs/source/getting_started/main_concepts.md

```diff
@@ -0,0 +1,124 @@
+# 🧠 Main Concepts
+
+## Overview
+Training a model with Streaming happens in 2 main steps:
+1. Convert your data to a StreamingDataset-compatible format
+2. Distributed model training with StreamingDataset
+
+Let's cover the key concepts in this process.
+
+## Dataset conversion
+Raw data samples need to be processed into a **stream**, or set of **shard** files, that allow for fast random access during training. Streaming supports the following file formats:
+* MDS (most performant)
+* XSV
```

Comment:
nobody has heard of XSV, but they'll recognize CSV and less so TSV

Reply:
changed to "csv/tsv"!